headless-server-commerce-base-lib/src/main/java/com/coremedia/blueprint/headlessserver/CommerceCaasAutoConfiguration.java

```diff
@@ -127,7 +127,7 @@
 import static java.lang.invoke.MethodHandles.lookup;
 import static java.util.Collections.emptyList;
 
-@AutoConfiguration(after = PluginManagerAutoConfiguration.class)
+@AutoConfigureAfter(PluginManagerAutoConfiguration.class)
```

Comment:
`@AutoConfigureAfter` should only be placed on auto configurations. You cannot just omit the `@AutoConfiguration` annotation, can you? Please check all usages of `@AutoConfigure[Before/After]` in this PR.

Reply:
You are right, it is just an alias for the "after" value of the AutoConfiguration annotation. I reverted the commit.